hardhat.config.ts

```diff
@@ -90,9 +87,8 @@ task('verify-sidekick')
     const block = await ethers.provider.getBlockNumber();
     const Sidekick = await ethers.getContractFactory('Sidekick');
     const sidekick = Sidekick.connect(signer).attach(args.sidekickAddr);
```

Comment:
```suggestion
    const sidekick = await ethers.getContractAt('Sidekick', args.sidekickAddr, signer);
```
I prefer it this way, but up to you.